lib/diagnostic.rb

```diff
@@ -141,5 +166,9 @@ def multiplier
     else
       1
     end
+
+    def get_possible_moves
+      puts @moves
+    end
```

Comment:
Because it comes after `private`, this method would be private, which was not specified.
Additionally, it's an instance method, not a class method.